x-pack/test/functional/page_objects/dataset_quality.ts

```diff
@@ -79,7 +79,7 @@ export function DatasetQualityPageObject({ getPageObjects, getService }: FtrProv
     datasetSearchInput: '[placeholder="Filter data sets"]',
     showFullDatasetNamesSwitch: 'button[aria-label="Show full data set names"]',
     showInactiveDatasetsNamesSwitch: 'button[aria-label="Show inactive data sets"]',
-    superDatePickerApplyButton: '.euiQuickSelect__applyButton',
+    superDatePickerApplyButton: '[data-test-subj="superDatePickerQuickSelectApplyButton"]',
```

Comment:
@cee-chen This is the line

Reply:
What exactly needs to be updated? That className was removed by EUI and replaced by a data-test-subj, so the highlighted line is correct.

Reply:
Does that mean that line 111, should be something like: https://github.com/elastic/kibana/blob/c6548c806d416dc79cd75c3aaf321f92674f2b8f/x-pack/plugins/security_solution/public/common/components/super_date_picker/index.test.tsx#L106 ?

Maybe I'm confused but I thought we'd want to actually use the data-test-subj attr syntax?

Reply:
We **are** using the the data-test-subj attr syntax. I'm not sure what branch you're looking at but this is what it is on this PR:

https://github.com/elastic/kibana/blob/ce200f68e17fe511822679fc498797731fc3d682/x-pack/plugins/security_solution/public/common/components/super_date_picker/index.test.tsx#L112

Reply:
FWIW CI would be failing if all instances of `.euiQuickSelect__applyButton` hadn't been updated in this PR, since that className no longer exists